### DISCUSSION TITLE
project update: `Inlang` ❤️ `pipeli18ne`

### DISCUSSION THREAD

**Discussion by ivanhofer** [pipeli18ne/RFC#26] [Announcements]
Hi,
I just want to give a quick update on this project:

First of all, I haven't really had the time (or took the time) to reach out to a lot of people to collect the feedback such a big product approach would need. But **I was able to reach a few people who care about internationalization**.

One of them was @samuelstroschein, the founder of [Inlang](https://inlang.com/) who also want's to make localization simpler by providing a variety of great tools. We had a productive call and we both realized that we have the same goals, and we are just approaching them from a different perspective.
I was able to convince him that a **great i18n "library" is needed to reach our goals**. And now I have the **opportunity to build that solution** inside the Inlang ecosystem.

This will probably mean that this project as it was outlined here will not happen. But **part of it will be implemented within Inlang** to build the **best i18n experience for the JavaScript ecosystem** 🎉.

So please head over to https://inlang.com/ to follow our progress. **We will have some good news to share in a few weeks!**

**Comment by samuelstroschein** [comment]
@ivanhofer it is great to have you on board for inlang!
